diff --git a/node_modules/ascii-table3/ascii-table3.styles.json b/node_modules/ascii-table3/ascii-table3.styles.json
index 2c38d97..e0f50fd 100644
--- a/node_modules/ascii-table3/ascii-table3.styles.json
+++ b/node_modules/ascii-table3/ascii-table3.styles.json
@@ -156,16 +156,16 @@
         "name": "ascii-girder",
         "borders": {
             "top": {
-                "left": "//", "center": "=", "right": "\\\\", "colSeparator": "[]"
+                "left": "´//", "center": "=", "right": "\\\\´", "colSeparator": "[]"
             },
             "middle": {
-                "left": "|]", "center": "=", "right": "[|", "colSeparator": "[]"
+                "left": "´|]", "center": "=", "right": "[|´", "colSeparator": "[]"
             },
             "bottom": {
-                "left": "\\\\", "center": "=", "right": "//", "colSeparator": "[]"
+                "left": "´\\\\", "center": "=", "right": "//´", "colSeparator": "[]"
             },
             "data" : {
-                "left": "||", "center": " ", "right": "||", "colSeparator": "||"
+                "left": "´||", "center": " ", "right": "||´", "colSeparator": "||"
             }
           }
     },
diff --git a/node_modules/ascii-table3/src/ascii-table3.js b/node_modules/ascii-table3/src/ascii-table3.js
index 7422b63..0e19625 100644
--- a/node_modules/ascii-table3/src/ascii-table3.js
+++ b/node_modules/ascii-table3/src/ascii-table3.js
@@ -39,7 +39,280 @@ class AsciiTable3 {
      */
     constructor(title = '') {
         /** @type {Style[]} */
-        this.styles = JSON.parse(fs.readFileSync(require.resolve(STYLES_FILENAME), 'utf8'))
+        this.styles = [
+            {
+                "name": "none",
+                "borders": {
+                    "top": {
+                        "left": "", "center": " ", "right": "", "colSeparator": " "
+                    },
+                    "middle": {
+                        "left": "", "center": " ", "right": "", "colSeparator": " "
+                    },
+                    "bottom": {
+                        "left": "", "center": " ", "right": "", "colSeparator": " "
+                    },
+                    "data" : {
+                        "left": "", "center": " ", "right": "", "colSeparator": " "
+                    }
+                }
+            },
+            {
+                "name": "compact",
+                "borders": {
+                    "top": {
+                        "left": "", "center": "-", "right": "", "colSeparator": "-"
+                    },
+                    "middle": {
+                        "left": "", "center": "-", "right": "", "colSeparator": " "
+                    },
+                    "bottom": {
+                        "left": "", "center": "", "right": "", "colSeparator": ""
+                    },
+                    "data" : {
+                        "left": "", "center": " ", "right": "", "colSeparator": " "
+                    }
+                }
+            },
+            {
+                "name": "ramac",
+                "borders": {
+                    "top": {
+                        "left": "+", "center": "-", "right": "+", "colSeparator": "+"
+                    },
+                    "middle": {
+                        "left": "+", "center": "-", "right": "+", "colSeparator": "+"
+                    },
+                    "bottom": {
+                        "left": "+", "center": "-", "right": "+", "colSeparator": "+"
+                    },
+                    "data" : {
+                        "left": "|", "center": " ", "right": "|", "colSeparator": "|"
+                    }
+                }
+            },
+            {
+                "name": "ascii-table",
+                "borders": {
+                    "top": {
+                        "left": ".", "center": "-", "right": ".", "colSeparator": "-"
+                    },
+                    "middle": {
+                        "left": "|", "center": "-", "right": "|", "colSeparator": "-"
+                    },
+                    "bottom": {
+                        "left": ".", "center": "-", "right": ".", "colSeparator": "-"
+                    },
+                    "data" : {
+                        "left": "|", "center": " ", "right": "|", "colSeparator": "|"
+                    }
+                }
+            },
+            {
+                "name": "ascii-reStructuredText",
+                "borders": {
+                    "top": {
+                        "left": "+", "center": "-", "right": "+", "colSeparator": "+"
+                    },
+                    "middle": {
+                        "left": "+", "center": "=", "right": "+", "colSeparator": "+"
+                    },
+                    "bottom": {
+                        "left": "+", "center": "-", "right": "+", "colSeparator": "+"
+                    },
+                    "data" : {
+                        "left": "|", "center": " ", "right": "|", "colSeparator": "|"
+                    }
+                }
+            },
+            {
+                "name": "ascii-reStructuredText-simple",
+                "borders": {
+                    "top": {
+                        "left": "=", "center": "=", "right": "=", "colSeparator": " "
+                    },
+                    "middle": {
+                        "left": "=", "center": "=", "right": "=", "colSeparator": " "
+                    },
+                    "bottom": {
+                        "left": "=", "center": "=", "right": "=", "colSeparator": " "
+                    },
+                    "data" : {
+                        "left": " ", "center": " ", "right": " ", "colSeparator": " "
+                    }
+                }
+            },
+            {
+                "name": "ascii-dots",
+                "borders": {
+                    "top": {
+                        "left": ".", "center": ".", "right": ".", "colSeparator": "."
+                    },
+                    "middle": {
+                        "left": ":", "center": ".", "right": ":", "colSeparator": ":"
+                    },
+                    "bottom": {
+                        "left": ":", "center": ".", "right": ":", "colSeparator": ":"
+                    },
+                    "data" : {
+                        "left": ":", "center": " ", "right": ":", "colSeparator": ":"
+                    }
+                }
+            },
+            {
+                "name": "ascii-rounded",
+                "borders": {
+                    "top": {
+                        "left": ".", "center": "-", "right": ".", "colSeparator": "."
+                    },
+                    "middle": {
+                        "left": ":", "center": "-", "right": ":", "colSeparator": "+"
+                    },
+                    "bottom": {
+                        "left": "'", "center": "-", "right": "'", "colSeparator": "'"
+                    },
+                    "data" : {
+                        "left": "|", "center": " ", "right": "|", "colSeparator": "|"
+                    }
+                  }
+            },
+            {
+                "name": "ascii-clean",
+                "borders": {
+                    "top": {
+                        "left": "", "center": "-", "right": "", "colSeparator": "|"
+                    },
+                    "middle": {
+                        "left": "", "center": "-", "right": "", "colSeparator": "|"
+                    },
+                    "bottom": {
+                        "left": "", "center": "-", "right": "", "colSeparator": "|"
+                    },
+                    "data" : {
+                        "left": "", "center": " ", "right": "", "colSeparator": "|"
+                    }
+                }
+            },
+            {
+                "name": "ascii-girder",
+                "borders": {
+                    "top": {
+                        "left": "´//", "center": "=", "right": "\\\\´", "colSeparator": "[]"
+                    },
+                    "middle": {
+                        "left": "´|]", "center": "=", "right": "[|´", "colSeparator": "[]"
+                    },
+                    "bottom": {
+                        "left": "´\\\\", "center": "=", "right": "//´", "colSeparator": "[]"
+                    },
+                    "data" : {
+                        "left": "´||", "center": " ", "right": "||´", "colSeparator": "||"
+                    }
+                  }
+            },
+            {
+                "name": "unicode-single",
+                "borders": {
+                    "top": {
+                        "left": "┌", "center": "─", "right": "┐", "colSeparator": "┬"
+                    },
+                    "middle": {
+                        "left": "├", "center": "─", "right": "┤", "colSeparator": "┼"
+                    },
+                    "bottom": {
+                        "left": "└", "center": "─", "right": "┘", "colSeparator": "┴"
+                    },
+                    "data" : {
+                        "left": "│", "center": " ", "right": "│", "colSeparator": "│"
+                    }
+                }
+            },
+            {
+                "name": "unicode-double",
+                "borders": {
+                    "top": {
+                        "left": "╔", "center": "═", "right": "╗", "colSeparator": "╦"
+                    },
+                    "middle": {
+                        "left": "╠", "center": "═", "right": "╣", "colSeparator": "╬"
+                    },
+                    "bottom": {
+                        "left": "╚", "center": "═", "right": "╝", "colSeparator": "╩"
+                    },
+                    "data" : {
+                        "left": "║", "center": "═", "right": "║", "colSeparator": "║"
+                    }
+                }
+            },
+            {
+                "name": "unicode-mix",
+                "borders": {
+                    "top": {
+                        "left": "╔", "center": "═", "right": "╗", "colSeparator": "╤"
+                    },
+                    "middle": {
+                        "left": "╟", "center": "─", "right": "╢", "colSeparator": "┼"
+                    },
+                    "bottom": {
+                        "left": "╚", "center": "═", "right": "╝", "colSeparator": "╧"
+                    },
+                    "data" : {
+                        "left": "║", "center": " ", "right": "║", "colSeparator": "│"
+                    }
+                }
+            },
+            {
+                "name": "unicode-round",
+                "borders": {
+                    "top": {
+                        "left": "╭", "center": "─", "right": "╮", "colSeparator": "┬"
+                    },
+                    "middle": {
+                        "left": "├", "center": "─", "right": "┤", "colSeparator": "┼"
+                    },
+                    "bottom": {
+                        "left": "╰", "center": "─", "right": "╯", "colSeparator": "┴"
+                    },
+                    "data" : {
+                        "left": "│", "center": " ", "right": "│", "colSeparator": "│"
+                    }
+                }
+            },
+            {
+                "name": "github-markdown",
+                "borders": {
+                    "top": {
+                        "left": "", "center": "", "right": "", "colSeparator": ""
+                    },
+                    "middle": {
+                        "left": "|", "center": "-", "right": "|", "colSeparator": "|"
+                    },
+                    "bottom": {
+                        "left": "", "center": "", "right": "", "colSeparator": ""
+                    },
+                    "data" : {
+                        "left": "|", "center": " ", "right": "|", "colSeparator": "|"
+                    }
+                }
+            },
+            {
+                "name": "reddit-markdown",
+                "borders": {
+                    "top": {
+                        "left": "", "center": " ", "right": "", "colSeparator": " "
+                    },
+                    "middle": {
+                        "left": "", "center": "-", "right": "", "colSeparator": "|"
+                    },
+                    "bottom": {
+                        "left": "", "center": "", "right": "", "colSeparator": ""
+                    },
+                    "data" : {
+                        "left": "", "center": " ", "right": "", "colSeparator": "|"
+                    }
+                }
+            }
+        ];        
 
         this.clear();
 
